tests/FakeItEasy.IntegrationTests/FakingClassesTests.cs

```diff
@@ -12,10 +11,13 @@ public class FakingClassesTests
         [Test]
         public void Should_be_able_to_get_a_fake_value_of_uri_type()
         {
-            using (Fake.CreateScope(new NullFakeObjectContainer()))
-            {
-                A.Fake<Uri>();
-            }
+            // Arrange
+
+            // Act
+            var fake = A.Fake<Uri>();
+
+            // Assert
+            Fake.GetFakeManager(fake).Should().NotBeNull("because we should be able to create a fake Uri");

```

Comment:
Oh, I see now. :blush: 

OK, I agree that the assertion can stay as is.

I do think we should change the test name (and probably use a custom type rather than `System.Uri`) and I also agree that that is out of scope here and should be a separate PR. I also think that the assertion reason should change to "because the object should be a fake".

I'll send the PR.